File: GSI QA testing tool/UI/UI_185_UnemploymentClaimConfirmation.cs
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GSI_QA_testing_tool.UI
{
    internal class UI_185_UnemploymentClaimConfirmation
    {
        public static void GoTo(IWebDriver driver)
        {
            if (Finder.FindIt(driver, "//input[@id='ctl00_Main_content_btnNext']"))
            {
                Debug.WriteLine("UnemploymentClaimConfirmation is On");
            }
            else
            {
                Debug.WriteLine("UnemploymentClaimConfirmation is Off");
                return;
            }

            Finder.ClickIt(driver, "//input[@id='ctl00_Main_content_btnNext']");
        }
    }
}

File: GSI QA testing tool/UI/UI_185_UnemploymentClaimConfirmation.vb
Imports OpenQA.Selenium
Imports System
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace GSI_QA_testing_tool.UI
    Friend Class UI_185_UnemploymentClaimConfirmation
        Public Shared Sub [GoTo](ByVal driver As IWebDriver, ByVal js As IJavaScriptExecutor)
            If Data._StopAt.Contains("Unemployment Claim Confirmation") Then
                Data.StopAtDialog()
            End If

            If Finder.FindItByCSS(driver, "*[id='ctl00_Main_content_btnNext']") Then
                Debug.WriteLine("UnemploymentClaimConfirmation is On")
            Else
                Debug.WriteLine("UnemploymentClaimConfirmation is Off")
                Throw New Exception("The claim is not completed")
            End If

            Finder.JSClickIt(driver, "#ctl00_Main_content_btnNext", js)
        End Sub
    End Class
End Namespace
